pkg/cephmgr/mon/info.go

```diff
@@ -77,6 +77,11 @@ func createOrGetClusterInfo(factory client.ConnectionFactory, etcdClient etcd.Ke
 
 // create new cluster info (FSID, shared keys)
 func CreateClusterInfo(factory client.ConnectionFactory, adminSecret string) (*ClusterInfo, error) {
+	return CreateNamedClusterInfo(factory, adminSecret, "rookcluster")
```

Comment:
that's only used for standalone rook where `rookcluster` is the default cluster name. Rook for k8s will use the namespace name